LGB.py

```diff
@@ -248,3 +253,9 @@ def eval_f(y_true,y_pred):
 # print(test['preds'].mean())
 test[['id','preds']].to_csv('submission/lgb_pred_test.csv', sep='\t',index=False,header=False)
 print(test['preds'])
+
+# Calculate score
+score = Evaluation.calculate_score(test,"data/testdata/test_df_label.csv","data/testdata/test_df.csv")
+print(("#Your score is %.4f")%(score*100.0))
+with open('ScoresLog.txt','a+') as scoreRecord:
```

Comment:
Suggestion: `'logs/LgbScoresLog.txt'`, also update `README.md` to add `mkdir logs`.

Reply:
Change to `Results/LGBScore.txt` and there is no need to add `mkdir `, because I have upload this folder.